app/views/events/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Events</h1>
+
+<%= link_to 'New Event', new_event_path %>
+
+<table>
+  <thead>
+    <tr>
+      <th>Event</th>
+    </tr>
```

Comment:
The event partial that you are loading has two `<td>` tags. I would expect there to be two `<th>` tags.